apps/payment-widgets/src/payment/index.css

```
@@ -169,7 +184,7 @@
 
 @utility block-child {
   & > * {
```

Stylelint (17.7.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)